v2/pkg/passive/sources.go

```diff
@@ -59,6 +60,7 @@ var DefaultSources = []string{
 	"riddler",
 	"securitytrails",
 	"shodan",
+	"sonarsearch",
```

Comment:
I think this source should be in the list of recursives, `DefaultRecursiveSources`, the rest is great!